pkg/utils/events.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+)
+
+type EventType string
+
+var (
+	EventTypeWarning EventType = "Warning"
+	EventTypeNormal  EventType = "Normal"
+)
+
+type EventSchema struct {
+	Events []DefaultEvent
+}
+
+type DefaultEvent struct {
```

Comment:
the names are confusing. maybe it is better to call this as `EventSchema` and the other one as `EventSchemaList`?